notebooks/LightGBM/README.md

```diff
@@ -0,0 +1,39 @@
+# Triton FIL LightGBM
+
+This notebook is a reference for deploying a forest model trained using LightGBM library in Triton Inference Server with Forest Inference Library (FIL) backend. The notebook explains how one can deploy LightGBM model in Triton, check deployment status and send inference requests, set concurrent model execution and dynamic batching and find the best deployment configuration using Model Analyzer.
+
+## Requirements
+* Nvidia GPU (T4 or V100 or A100)
```

Comment:
Sure, What other features can we highlight for Triton FIL with this example?

Reply:
I'm not sure, actually. I can't think of anything specific to FIL that would be good to highlight except little things like `predict_proba` or something like that. This would be more if there's a Triton feature we want to highlight. Anything you can think of that would be good to advertise? Otherwise, we can just trim back the overall size of the example.